052-record-reconciled-versions.md

```diff
@@ -1,18 +1,19 @@
 # Record Reconciled Version in Kafka Custom Resource status
 
-For the `Kafka` Custom Resource it is often unclear when an upgrade has finished, whether all the brokers have rolled and to which version of the operator a cluster has been reconciled with.
-The following proposal puts forward a mechanism to make this clear in the `Kafka` custom resources status, making it easy to programmatically check for upgrade completion that could also be used/adopted in the upgrade system tests, simplifying the code and making for a better upgrade UX story.
+For the `Kafka` Custom Resource it is often unclear when an upgrade has truly finished, whether all the brokers have rolled and to which version of the operator a cluster has been reconciled with and which version of Kafka.
+The following proposal puts forward a mechanism to make it clear in the `Kafka` custom resources status whether an upgrade is complete, making it easy to programmatically check for upgrade completion. This could potentially be used/adopted in the upgrade system tests or upgrade verification steps for a user, simplifying the code and making for a better upgrade UX story.
 
 ## Current situation
 
 Currently the only way to know if an upgrade has finished to completion is to check is all the zookeeper and kafka brokers have rolled.
 https://github.com/strimzi/strimzi-kafka-operator/blob/2a1fdf9d8695bb22a2bf977b0ba5414291530207/systemtest/src/test/java/io/strimzi/systemtest/upgrade/KafkaUpgradeDowngradeST.java#L292
+Additionally, if a user updates the Kafka version, a user needs to check if their Kafka version has been updated by inspecting the Kafka protocol version that the Brokers are running with, and this could require two rolls of the brokers, not just a single roll.
```

Comment:
If a user has the `inter.broker.protocol.version` config setting set to the old value, would they still be able to see the new Kafka version?